src/main/java/duke/Parser.java

```diff
@@ -11,6 +11,7 @@
     private static final String CMD_DDL = "deadline";
     private static final String CMD_EVENT = "event";
     private static final String CMD_DEL = "delete";
+    private static final String CMD_FIND = "find";
```

Comment:
Do consider making your variable declaration clearer, in a sense that people who are reading your code may not understand what CMD is referring to, or what DDL is referring to. For example, COMMAND_DEADLINE or something like that